### PR TITLE
[docs] Describe how to enable a fuzzer for a job on the Job page.

### PR DESCRIPTION
**Pull request by Dor1s**
This makes the doc and the process simpler. The feature was added in https://github.com/google/clusterfuzz/pull/1901

### CODE REVIEW

File: docs/setting-up-fuzzing/libfuzzer_and_afl.md [hunk 1]
```diff
@@ -100,23 +100,23 @@ To create a job for libFuzzer or AFL:
...
 2. Go to the "ADD NEW JOB" form.
 3. Fill out the "Name" and "Platform" (LINUX).
-  1. If setting up an **AFL** job, use the templates **"afl"** and
-     **"engine_asan"**.
-  2. If setting up a **libFuzzer** job, use the templates **"libfuzzer"** and
-     **"engine_$SANITIZER"** depending on which sanitizer you are using (e.g.
-     **"engine_asan"**).
-4. Select your build (your zip containing the fuzz target binary) to upload as a
+4. Enable the desired fuzzer in the "Select/modify fuzzers" field, e.g. **libFuzzer** or **afl**.
+5. If setting up an **AFL** job, use the templates **"afl"** and
```
Comment (Dor1s) [inline docs/setting-up-fuzzing/libfuzzer_and_afl.md:104]: not sure why we had the indentation here, it is rendered as a straight list anyway: https://google.github.io/clusterfuzz/setting-up-fuzzing/libfuzzer-and-afl/#creating-a-job-type

File: docs/setting-up-fuzzing/libfuzzer_and_afl.md [hunk 2]
```diff
@@ -100,23 +100,23 @@ To create a job for libFuzzer or AFL:
...
-
-Next we must let ClusterFuzz know which fuzzer the job can be used with:
-1. Navigate to the *Fuzzers* page.
-2. Click "EDIT" for the desired fuzzer (afl or libFuzzer).
-3. Click "Select/modify jobs".
-4. Mark the desired job.
-5. Click "SUBMIT".
+8. Use the "ADD" button to add the job to ClusterFuzz.
+
+You can also control which fuzzer the job can be used with using the *Fuzzers* page:
```
Comment (oliverchang) [inline docs/setting-up-fuzzing/libfuzzer_and_afl.md:115]: I would actually just remove this completely to keep the documentation lean. I think it's actually quite uncommon to want to modify associations from the fuzzer page instead.

**Review by oliverchang — APPROVED** [review]
thanks!

File: docs/setting-up-fuzzing/libfuzzer_and_afl.md (hunk 2 above)
Comment (inferno-chromium) [reply docs/setting-up-fuzzing/libfuzzer_and_afl.md:115]: I am fine with either. Can we expand this page to include honggfuzz as well.

**Review by inferno-chromium — APPROVED** [review]

File: docs/setting-up-fuzzing/libfuzzer_and_afl.md (hunk 2 above)
Comment (Dor1s) [reply docs/setting-up-fuzzing/libfuzzer_and_afl.md]: cool, I was also thinking maybe we should remove it -- done!

also added a step for honggfuzz

**Pull request merged**